### DISCUSSION TITLE
Is there some way to specify an alphabetical range for downloads?

### DISCUSSION THREAD

**Discussion by Sargon-Aelther** [Kalanyr/gogrepo#90] [Q&A]
Hi there,

I am about to reach the limit of my hard drive and I'm wondering if there is some way to split volumes up? For example download only IDs starting with A-M and a separate one for N-Z?

Please don't give me advice about RAID. Thank you.

**Comment by Kalanyr** [comment]
Not inherently I'm afraid.

You can manually generate a list of titles from the folder names and feed
it into -ids or -skipids though.

Or you can use -skipgames and -skipextras to split extras and installers
onto separate drives.

**Reply by Sargon-Aelther** [reply]
Thanks. What is the maximum character count for a single command? Because my library is over 3000, so having ~1000 IDs in a single line seems a bit much and unrealistic.

Splitting off extras may work for a the short term (I'm already splitting by OS), but it's not going to help for long. Having some way to skip patches would contribute more than splitting extras, as some patches are very inefficient, essentially containing the whole game... and there's often multiple version of those, thus taking up maybe 3 times the space than the full game lol.
